salt/orchestrate/edx/init.sls

```diff
@@ -16,13 +16,12 @@ generate_edx_cloud_map_file:
         business_unit: {{ BUSINESS_UNIT }}
         environment_name: {{ ENVIRONMENT }}
         purpose_prefix: {{ PURPOSE_PREFIX }}
-        roles:
-          - edx
-          - log-forwarder
         securitygroupid:
           - {{ salt.boto_secgroup.get_group_id(
               'edx-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
           - {{ salt.boto_secgroup.get_group_id(
+              'edx-worker{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
```

Comment:
This will be applied to all of the nodes in the map template. You might want to break out the different groups into individual ids in the context object and then compile the list of groups via the named references in the map template.